src/meta/processors/partsMan/GetPartsAllocProcessor.cpp

```diff
@@ -34,9 +34,65 @@ void GetPartsAllocProcessor::process(const cpp2::GetPartsAllocReq& req) {
     }
     handleErrorCode(nebula::cpp2::ErrorCode::SUCCEEDED);
     resp_.set_parts(std::move(parts));
+    auto terms = getTerm(spaceId);
+    if (!terms.empty()) {
+        resp_.set_terms(std::move(terms));
+    }
     onFinished();
 }
 
+std::unordered_map<PartitionID, TermID> GetPartsAllocProcessor::getTerm(GraphSpaceID spaceId) {
```

Comment:
A suggestion，is better to change the argument to `const vector<std::string>& keys`? this will reduce a database scan of `MetaServiceUtils::partPrefix(spaceId)`.